Product/Production/Deploy/ear/weblogic/pom.xml

```diff
@@ -19,11 +19,6 @@
         </dependency>
         <dependency>
             <groupId>org.apache.geronimo.specs</groupId>
-            <artifactId>geronimo-jaxws_2.2_spec</artifactId>
```

Comment:
In other Application servers this jar was added by cxf bundle. so the exclusion from cxf-bundle helped to remove this jar. But in Weblogic this jar is defined as a dependency in Weblogic specific pom.xml. Also it was tested by deploying the ear to Weblogic Application server.